contrib/mesos/docs/issues.md

```diff
@@ -198,6 +198,16 @@ The scheduler also offers `/debug` API endpoints that may be useful:
 All of the issues in the above section also apply to the Kubernetes-Mesos DCOS package builds.
 The issues listed in this section apply specifically to the Kubernetes-Mesos DCOS package available from https://github.com/mesosphere/multiverse.
 
+### Etcd
+
+The default configuration of the DCOS Kubernetes package launches an internal etcd process **which only persists the cluster state in the task's sandbox**. While this is simpler for the first steps with Kubernetes-Mesos, it means that any cluster state is lost when the Kubernetes-Mesos Docker container is restarted.
```

Comment:
"which only persists the cluster state in the sandbox of the current container instance"